src/com/serotonin/mango/vo/report/ReportChartCreator.java

```diff
@@ -147,8 +148,9 @@ else if (pointStat.getDiscreteTimeSeries() != null)
                 // The path comes from the servlet path definition in web.xml.
                 model.put("chartName", IMAGE_SERVLET + chartName);
             }
+            SystemSettingsService systemSettingsService = new SystemSettingsService();
             int consolidatedChartHeight = ImageChartUtils.calculateHeightChart(pointStatistics, IMAGE_HEIGHT_PIXELS,
-                    POINT_LABEL_HEIGHT_IN_LEGEND_PIXELS, LINE_LENGTH_IN_LEGEND_LIMIT, DATA_POINT_EXTENDED_NAME_LENGTH_LIMIT);
+                    POINT_LABEL_HEIGHT_IN_LEGEND_PIXELS, LINE_LENGTH_IN_LEGEND_LIMIT, systemSettingsService.getPointNamesLengthInReport());
```

Comment:
We change all parameter variable names according to this pattern: DATA_POINT_EXTENDED_NAME_LENGTH_LIMIT 

If you move a variable to the configuration, leave the name of this parameter, adding only the context, i.e. Report.

systemSettingsService.getDataPointExtendendNameLengthInReportLimit()
systemSettingsService.setDataPointExtendendNameLengthInReportLimit()